### PR TITLE
docs - new gp_toolkit views for missing/orphaned files

### PR DESCRIPTION
**Pull request by lisakowen**
docs for https://github.com/greenplum-db/gpdb/pull/15161.

new gp_toolkit views to identify missing and orphaned data files.

doc review site link (behind vpn):  https://docs-staging.vmware.com/en/draft/VMware-Greenplum/datafile-views/greenplum-database/GUID-ref_guide-gp_toolkit.html#checking-for-missing-and-orphaned-data-files

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/gp_toolkit.html.md [hunk 1]
```diff
@@ -890,6 +892,58 @@ This external table runs the `df` \(disk free\) command on the active segment ho
...
+
+The `gp_check_orphaned_files` view scans the default and user-defined tablespaces for orphaned data files. Greenplum Database considers both normal data files and extended numbered files (files that contain a `.<N>` in the name) in this check.
+
+|Column|Description|
+|------|-----------|
+| gp_segment_id | The Greenplum Database segment identifier. |
+| tablespace | The identifier of the tablespace in which the orphaned file resides. |
+| filename | The file name of the orphaned data file. |
+
+`gp_check_orphaned_files` gathers results from the Greenplum Database coordinator and all segments.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/gp_toolkit.html.md:919]: For each of these views, it seems like this sentence should be included in the preceding paragraph (before the table reference).

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/markdown/ref_guide/gp_toolkit.html.md [hunk 2]
```diff
@@ -890,6 +892,55 @@ This external table runs the `df` \(disk free\) command on the active segment ho
 |dfdevice|The device name|
 |dfspace|Free disk space in the segment file system in kilobytes|
 
+## <a id="missingfiles"></a>Checking for Missing and Orphaned Data Files
+
+Greenplum Database considers a relation data file that is present in the catalog, but not on disk, to be missing. Conversely, when Greenplum encounters an unexpected data file on disk that is not referenced in any relation, it considers that file to be orphaned.
+
+Greenplum Database provides the following views to help identify if missing or orphaned files exist in your cluster:
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/gp_toolkit.html.md:899]: nit: the views only check objects in the current database, so let's add that info here.

**Review by huansong — APPROVED** [review]
LGTM! Thanks for adding these.

**Comment by lisakowen** [comment]
thank you for reviewing, @huansong

**Pull request merged**